benchmark/CMakeLists.txt

```diff
@@ -50,15 +50,23 @@ macro(add_dataset datasets dataset path_to_template range)
 endmacro()
 
 add_custom_target(benchmarks)
-macro(add_chart target)
+macro(add_chart type case)
     cmake_parse_arguments(ARGS "" "" "DATASETS" ${ARGN})
 
+    set(type_target "benchmarks.${type}")
```

Comment:
My reasoning is that it makes sense to think of target names in this context in analogy to filesystem paths, i.e. the longer the prefix, the more specific it is, up to the point where it denotes a single entity. Notice that `benchmarks.hetero.at` is really just the same thing as `documents/research/some_report.txt`.